networkx/algorithms/tournament.py

```diff
@@ -368,10 +374,15 @@ def is_strongly_connected(G):
     Examples
     --------
     >>> from networkx.algorithms import tournament
-    >>> G = nx.DiGraph([(0, 1), (0, 2), (0, 3), (1, 2), (1, 3), (2, 3), (3, 0)])
+    >>> G = nx.DiGraph([(0, 1), (0, 2), (1, 2), (1, 3), (2, 3), (3, 0)])
+    >>> tournament.is_tournament(G)
+    True
     >>> tournament.is_strongly_connected(G)
     True
-    >>> G.remove_edge(1, 3)
+    >>> G.remove_edge(3,0)
+    >>> G.add_edge(0,3)
```

Comment:
Thanks -- this is good.
I did find one place where the style guide would like a space after the comma inside a function call (these two lines).  Could you make that change and then push to your github branch -- it will automatically update the PR.